packages/@vuepress/theme-default/components/DropdownLink.vue

```diff
@@ -75,13 +76,18 @@ export default {
   },
 
   props: {
+    inSidebar: {
```

Comment:
It'll only display in sidebar if the device is mobile, so it seems that you only need one of them (`inSidebar` or `isMobile`).